src/main/java/com/immunopass/repository/OrderRepository.java

```diff
@@ -1,8 +1,19 @@
 package com.immunopass.repository;
 
-import org.springframework.data.jpa.repository.JpaRepository;
 import com.immunopass.entity.OrderEntity;
+import com.immunopass.enums.OrderStatus;
+import com.immunopass.model.Order;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 
 
 public interface OrderRepository extends JpaRepository<OrderEntity, Long> {
+
+    @Modifying
+    @Query("update voucher set status = :status where order_id = :orderID")
```

Comment:
See if we can do same without writing the query.

Reply:
Query just makes it clearer for me.